apps/omg_conformance/test/support/conformance/merkle_proofs.ex

```diff
@@ -49,8 +50,25 @@ defmodule Support.Conformance.MerkleProofs do
     # only geth is supported for the merkle proof conformance tests for now
     :geth = Application.fetch_env!(:omg_eth, :eth_node)
 
-    # revert from `Eth.call_contract` it returns something resembling a reason
+    # revert from `call_contract` it returns something resembling a reason
     # binary (beginning with 4-byte function selector). We need to assume that this is in fact a revert
     assert <<0::size(28)-unit(8)>> = binary_part(chopped_reason_binary_result, 4, 28)
   end
+
+  defp call_contract(contract, signature, args, return_types) do
```

Comment:
this is the only use case for `"checkMembership(bytes,uint256,bytes32,bytes)"` contract call.because of this, it does not belong in production code.